AIMSICD/src/main/res/values/translatable_strings.xml

```diff
@@ -42,8 +42,8 @@
     <string name="device_info_title">Device Information</string>
     <string name="sim_info_title">SIM Information</string>
     <string name="network_info">Network Information</string>
-    <string name="cell_info_title">Neighboring Cells</string>
-    <string name="neighboring_cells_title">Neighboring Cells</string>
+    <string name="cell_info_title">Neighbouring Cells</string>
```

Comment:
@larsgrefer, how come you're re-introducing neighb**ou**ring? I aimed for *one* unified word: `neighboring`.